### DISCUSSION TITLE
What ports are used by modoboa?

### DISCUSSION THREAD

**Discussion by Dr-Ari-Gami** [modoboa/modoboa#2723] [Help] [answered]
I'd like to configure UFW, and can't seem to find the list of ports that modoboa uses. 

Does anyone know what ports I need to whitelist?

Thanks!

**Comment by MrGeneration** [comment]
Modoboa specifically doesn't use any external ports, the third party tools it uses do.
The easiest way to find a list of ports that may be relevant would be using `ss -tulpen`.

The port range depends on your configuration but normally would at least contain http(s), imap(s) and smtp(s).

**Comment by Dr-Ari-Gami** [comment, marked as answer]
Thanks! For future reference, I ended up whitelisting.

HTTP(s): 80, 443
SMTP: 25, 587, 465
IMAP: 143, 993

**Reply by shanmugad** [reply]
POP3? 

Modoboa do not support POP3?

**Reply by Dr-Ari-Gami** [reply]
Not to my knowledge. I found nothing running on ports 110 and 995.

Edit: looks like it can be enabled in the installer config. See #2008

**Reply by MrGeneration** [reply]
You may want to consider not to allow IMAP without encryption (Port 143).
Not opening said port on your firewall is enough.

This is a potential security risk. Don't block Port 25 though, it's required inbound.
